@xen-orchestra/lite/src/components/host/system/HostSystemGeneralInformation.vue

```diff
@@ -3,31 +3,25 @@
     <UiTitle>
       {{ t('general-information') }}
     </UiTitle>
-    <VtsQuickInfoRow :label="t('name')" :value="host.name_label" />
-    <VtsQuickInfoRow :label="t('uuid')" :value="host.uuid" />
-    <VtsQuickInfoRow :label="t('description')" :value="host.name_description" />
-    <VtsQuickInfoRow :label="t('tags')">
-      <template v-if="host.tags.length > 0" #value>
-        <UiTagsList class="value">
-          <UiTag v-for="tag in host.tags" :key="tag" accent="info" variant="secondary">
-            {{ tag }}
-          </UiTag>
-        </UiTagsList>
-      </template>
-    </VtsQuickInfoRow>
-    <VtsQuickInfoRow :label="t('status')">
+    <UiLabelValue :label="t('name')" :value="host.name_label" />
+    <UiLabelValue :label="t('uuid')" :value="host.uuid" />
+    <UiLabelValue :label="t('description')" :value="host.name_description" />
+    <UiLabelValue :label="t('tags')" :value="host.tags" />
+    <UiLabelValue :label="t('status')">
       <template #value>
         <VtsStatus :status="host.enabled" />
       </template>
-    </VtsQuickInfoRow>
-    <VtsQuickInfoRow :label="t('pool')">
+    </UiLabelValue>
+    <UiLabelValue :label="t('pool')">
       <template v-if="pool !== undefined" #value>
-        <UiLink size="medium" :to="`/pool/${pool.uuid}/`" icon="fa:city">
-          {{ pool.name_label }}
+        <UiLink size="medium" :to="`/pool/${pool.uuid}/`" icon="fa:city" class="link">
```

Comment:
```suggestion
        <UiLink size="medium" :to="`/pool/${pool.uuid}/`" icon="fa:city">
```
I don't think this class is usefull

Reply:
If we remove this class, we have an overflow, so before an update of `uiLink`, I think we can manage this in the page.
<img width="526" height="72" alt="image" src="https://github.com/user-attachments/assets/0184d32c-8b07-4f93-b82f-e4a80cce8a04" />

<img width="531" height="77" alt="image" src="https://github.com/user-attachments/assets/eace7307-7513-4fbc-949e-ca054a1756f5" />